src/App.tsx

```diff
@@ -15,71 +18,84 @@ export const goods = [
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handlerAddGood = (good: string) => this.setState({ selectedGood: good });
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  handlerClearGood = () => this.setState({ selectedGood: '' });
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  render() {
+    const {
+      selectedGood,
+    } = this.state;
 
-        <tr data-cy="Good">
-          <td>
+    return (
+      <main className="section container">
+        <h1
+          className={selectedGood ? 'title is-flex is-align-items-center' : 'title'}
+        >
+          {selectedGood ? `${selectedGood} is selected` : 'No goods selected' }
+          {selectedGood && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={this.handlerClearGood}
+            />
+          )}
+        </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+        <table className="table">
+          <tbody>
+            {goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={(selectedGood === good) ? 'has-background-success-light' : ''}
+              >
+                {selectedGood === good
+                  ? (
+                    <td>
+                      <button
+                        data-cy="RemoveButton"
+                        type="button"
+                        className={(selectedGood === good) ? 'button is-info' : 'button'}
```

Comment:
use classnames library

Reply:
:arrow_up: it is already installed by `npm`, you need just import it